language/move-vm/types/src/gas_schedule.rs

```diff
@@ -21,6 +23,8 @@ use vm::{
     file_format_common::{instruction_key, Opcodes},
 };
 
+pub const MAX_TRANSACTION_SIZE_IN_BYTES: usize = 4096;
```

Comment:
This feels like it should eventually be a parameter of `CostTable` to allow customization. But moving it here + killing the dep on `libra_types` is a great step in the right direction for now.

Reply:
This constant is already held in the `CostTable` :) In fact, as I was looking at it, there are a number of constants defined in that file already related to gas/transaction size, so that seems like a better place for this than here. 

Could you possibly move this constant to `move-core/types/src/gas_schedule.rs` and add it in with the other constant definitions there?

Reply:
yeah I like this and removing the dependency is great! thanks!

Reply:
> This constant is already held in the `CostTable` in the `GasConstants` struct :) In fact, as I was looking at it, there are a number of constants defined in that file already related to gas/transaction size, so that seems like a better place for this than here.
> 
> Could you possibly move this constant to `move-core/types/src/gas_schedule.rs` and add it in with the other constant definitions there?

Updated. Now, it lives in move-core-types. Please take a look.